src/components/card/FlipCard.js

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from 'styled-components/macro'
+import { useState } from 'react'
+import mainBackground from '../../assets/images/mainBackground.jpg'
+
+export default function FlipCard(props) {
+  const [isCardFlipped, setIsCardFlipped] = useState(false)
+  const isFreeOfCharge = props.freeOfCharge
+  const handleFlipCardClick = () => {
+    setIsCardFlipped(!isCardFlipped)
+  }
+  return (
+    <Card onClick={handleFlipCardClick}>
+      <CardContent isCardFlipped={isCardFlipped}>
+        <CardFront>
+          <CardTitle>{props.name}</CardTitle>
+          <CardSubtitle>{props.city}</CardSubtitle>
+        </CardFront>
+        <CardBack>
+          <CardBody>
+            <p>
+              <b>Name:</b> {props.name}
+            </p>
+            <br />
+            <p>
+              <b>Description:</b> {props.description}
+            </p>
+            <br />
+            <p>
+              <b>Address:</b> {props.street}
+              <br />
+              {props.zipCode} {props.city}, {props.country}
+            </p>
+            <br />
+            <p>
+              <b>Opening Hours:</b> {props.openingHours}
+            </p>
+            <br />
+            <p>
+              <b>Website:</b>{' '}
+              <a
+                href={props.website}
+                target="_blank"
+                rel="noreferrer"
+                onClick={event => event.stopPropagation()}
```

Comment:
You got a bit confused here. A `a` (anchor) element doesn't get an `event` on an `onClick`. This line does nothing and should be deleted.
```suggestion
```

Reply:
Ah, never mind. It stops the card from flipping, right?